hub/core/meta/tensor_meta.py

```diff
@@ -144,6 +156,7 @@ def _required_meta_from_htype(htype: str) -> dict:
         "min_shape": [],
         "max_shape": [],
         "length": 0,
+        "compression": defaults["compression"],
```

Comment:
```suggestion
        "default_compression": defaults["default_compression"],
```

this may seem weird, but it's important because `default_compression` tells the user that if no compression is being used, it will use `default_compression`

Reply:
```suggestion
        "default_compression": defaults.get("default_compression", None),
```

reason being: if `defaults` doesn't contain a `compression`, `_remove_none_values_from_dict` will handle removing it